LobbyServer2/LobbyServer/Matchmaking/MatchmakingQueue.cs

```diff
@@ -57,6 +57,8 @@ public LobbyMatchmakingQueueInfo AddGroup(long groupId, out bool added)
         {
             added = QueuedGroups.TryAdd(groupId, 0);
             MatchmakingQueueInfo.QueuedPlayers = GetPlayerCount();
+            MatchmakingQueueInfo.AverageWaitTime = TimeSpan.FromSeconds(0);
+            MatchmakingQueueInfo.QueueStatus = ServerManager.IsAnyServerAvailable() ? QueueStatus.WaitingForHumans : QueueStatus.AllServersBusy;
```

Comment:
Probably it should be the other way around, we tell that there are no servers only after we have got enough players for a game

Reply:
Cannot update with MatchmakingQueueStatusNotification if we do not have a gametype yet client won't allow it and at this stage we do not have this yet we cannot double use MatchmakingQueueAssignmentNotification without throwing a player out of queue first